ListingManager.Tests/FileManagerTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace ListingManager.Tests
+{
+    [TestClass]
+    public class FileManagerTests
+    {
+
+        [TestMethod]
+        public void GetChapterNumber()
+        {
+            string chapter1 = "Chapter01";
+            string chapter42 = "asdasdsad/asd/asd/asd/Chapter42";
```

Comment:
In best practice, every test should have one "logical" assert. We're testing two different test cases here, so we should make this two separate tests, or preferably, use DataRows. Look at the ListingInformation tests for an example of this.